src/main/scala/cromiam/samiam/SamIamActor.scala

```diff
@@ -0,0 +1,56 @@
+package cromiam.samiam
+
+import akka.actor.{Actor, ActorLogging, Props}
+import cromiam.samiam.SamIamActor._
+
+import scala.concurrent.ExecutionContext
+import scala.util.Random
+import scala.concurrent.duration._
+
+final class SamIamActor(allowedUsers: List[String]) extends Actor with ActorLogging {
```

Comment:
Hmm, good point.

I don't know what it should be, but probably not "everything's a child of 'system' and if it crashes then so does cromiam".

I'll add a ticket to sort this out